docs/Video Streaming for Navigation Apps/Introduction/index.md

```diff
@@ -99,4 +99,36 @@ func hmiLevel(_ oldLevel: SDLHMILevel, didChangeToLevel newLevel: SDLHMILevel) {
 To present a keyboard (such as for searching for navigation destinations), you should use the @![iOS]`SDLScreenManager`!@@![android]`ScreenManager`!@'s keyboard presentation feature. For more information, see the [Popup Menus and Keyboards](Displaying a User Interface/Popup Menus and Keyboards) guide.
 
 ## Navigation Subscription Buttons
-Head units supporting RPC v6.0+ may support navigation-specific subscription buttons for the navigation template. These subscription buttons allow your user to manipulate the map using hard buttons located on car's center console or steering wheel. It is important to support these subscription buttons in order to provide your user with the expected in-car navigation user experience. This is especially true on head units that don't support touch input as there will be no other way for your user to manipulate the map. See [Template Subscription Buttons](Displaying a User Interface/Template Subscription Buttons) for a list of these navigation buttons.
\ No newline at end of file
+Head units supporting RPC v6.0+ may support navigation-specific subscription buttons for the navigation template. These subscription buttons allow your user to manipulate the map using hard buttons located on car's center console or steering wheel. It is important to support these subscription buttons in order to provide your user with the expected in-car navigation user experience. This is especially true on head units that don't support touch input as there will be no other way for your user to manipulate the map. See [Template Subscription Buttons](Displaying a User Interface/Template Subscription Buttons) for a list of these navigation buttons.
+
+## Canceling the Route
```

Comment:
It think we already have a lot of random stuff getting adding to the bottom of the introduction section so it is starting to seem a little hodge-podgy. I _think_ when the issue refers to "route" it means the turn-by-turn related RPCS? If so, maybe this paragraph should go in the **Video Streaming for Navigation Apps > Displaying Turn Directions** section. If it refers to something else then I think it should go in the app services section.

Reply:
@NicoleYarroch I believe routes is referring to the developers application and the goal is to tell the developer to implement a listener for when the navigation service goes inactive. I do agree that the Video Streaming page is a little cluttered but I think the `Creating an App Service` page to be more appropriate

Reply:
Correct, the routes are the developer's application routing to a location. This definitely should not be moved to `Displaying Turn Directions`. App Services is appropriate, however, no nav apps currently implement app services, and this should be an indicator that they ought to (1) support app services, and (2) end their app's routing when their app service becomes inactive.